app/src/main/res/xml/visual_settings.xml

```diff
@@ -62,19 +62,19 @@
                 android:title="@string/font_type"
                 android:entries="@array/preferences_interface_general_icons_text_font_family_entries"
                 android:entryValues="@array/preferences_interface_general_icons_text_font_family_values"
-                android:defaultValue="sans-serif-condensed"/>-->
+                android:defaultValue="sans-serif-condensed"/>
 
         <CheckBoxPreference
             android:key="show_search_bar"
             android:title="@string/show_search_bar"
             android:defaultValue="true"
             />
 
-        <!--<CheckBoxPreference
+        <CheckBoxPreference
             android:key="show_page_indicator"
             android:title="@string/show_page_indicator"
             android:defaultValue="true"
             />-->
```

Comment:
This end-comment tag (`-->`) needs to be moved to after the `</PreferenceCategory>` this was breaking the build.

I would rather not disable this setting at all, though. I put in a comment above, that talks how we could continue using the setting to hide the search bar.